src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -89,19 +89,19 @@ internal T WithHttpManager(IHttpManager httpManager)
         /// was already set on the application builder
         public T WithLoggingCallback(
             LogCallback loggingCallback, 
-            LogLevel logLevel = LogLevel.Info, 
-            bool enablePiiLogging = false, 
-            bool enableDefaultPlatformLogging = false)
+            LogLevel? logLevel = null, 
+            bool? enablePiiLogging = null, 
```

Comment:
Why not use defaults as your original commit?

Reply:
If someone does a CreateWithOptions (which they may want to do if loading settings out of json) then there's no way to get them available when actually wiring up the callback (which you can't do from config).  This makes sure that we only look at the values passed in if someone set them to something besides null and then we can use the value, otherwise, fallback to what was already there.